lib/events/playback.go

```diff
@@ -70,6 +71,57 @@ func DetectFormat(r io.ReadSeeker) (*Header, error) {
 	return &Header{Tar: true}, nil
 }
 
+// Export exports session file events to json, text or yaml
+func Export(ctx context.Context, rs io.ReadSeeker, w io.Writer, exportFormat string) error {
+	switch exportFormat {
+	case teleport.Text:
```

Comment:
Looks like only `json` is supported at the moment.  Remove references to `text`/`yaml`.